src/main/java/mate/academy/springboot/datajpa/controller/CategoryController.java

```diff
@@ -0,0 +1,60 @@
+package mate.academy.springboot.datajpa.controller;
+
+import jakarta.validation.Valid;
+import mate.academy.springboot.datajpa.dto.CategoryRequestDto;
+import mate.academy.springboot.datajpa.dto.CategoryResponseDto;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import mate.academy.springboot.datajpa.service.mapper.RequestMapperDto;
+import mate.academy.springboot.datajpa.service.mapper.ResponseMapperDto;
+import org.springframework.web.bind.annotation.DeleteMapping;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.PutMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/category")
+public class CategoryController {
+    private final CategoryService categoryService;
+    private final RequestMapperDto<CategoryRequestDto, Category> categoryRequestMapperDto;
+    private final ResponseMapperDto<CategoryResponseDto, Category> categoryResponseMapperDto;
```

Comment:
There is the same implementation for both interfaces. Better have one interface DtoMapper with two methods.